components/keyrings/common/data/data.h

```diff
@@ -24,13 +24,68 @@
 #define DATA_INCLUDED
 
 #include <string>
+#include "global_default_mr.h"
 
 namespace keyring_common {
 namespace data {
 
 /** Data types */
-using Type = std::string;
-using Sensitive_data = std::string;
+using Type = pmr_string;
```

Comment:
May be, if we are targeting 8.0.27+, we can use `std::pmr::string` from `c++17` and forget about boost here

Reply:
I'll check if that works on all platfroms, as the boost version had issues with old centos.

Reply:
It's not possible - older platforms support C++17, but only std::experimental::pmr which isn't completely implemented.

Reply:
Yeah, I understand :(